massgov/openmass
# DP-22655: Migrate org page Featured Items Mosaic to sections

**Description:**
Added logic to migrate featured items data for org_pages to sections.

**Jira:** (Skip unless you are MA staff)
https://massgov.atlassian.net/browse/DP-22655


**To Test:**
- Visit an org_page that had Featured Items content.
- [ ] Verify the content it now displayed on the page.
- Login and edit the org_page.
- [ ] Verify the Featured Items in the Organization Sections field is the same as the one in the Featured Items field.
- [ ] Verify the org_page revision didn’t update.

Feature3 comparison links:
- Feature3: https://massgovfeature3.prod.acquia-sites.com/orgs/division-of-local-services
- Prod: Division of Local Services 

**Screenshots/GIFs:**

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
